addons/test_mail/tests/test_performance.py

```diff
@@ -816,7 +816,7 @@ def test_complete_message_post(self):
         ]
         self.attachements = self.env['ir.attachment'].with_user(self.env.user).create(self.vals)
         attachement_ids = self.attachements.ids
-        with self.assertQueryCount(emp=92 + 17):
+        with self.assertQueryCount(emp=98 + 17):
```

Comment:
If it goes from 92 to 98 (+6), and there are 3 partners to notify on message_post, then each partner to notify increases query count by 2 with your changes, I guess?

Could you enlighten on this change? Thanks!

Reply:
yes, one of them is for channel_partner_ids and second one is for user_ids because of these two lines https://github.com/odoo/odoo/pull/36000/files#diff-debf30f858f349749391b9d662b7eedaR2193-R2194